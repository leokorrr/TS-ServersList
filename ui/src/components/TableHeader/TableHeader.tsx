import React from 'react'

const TableHeader: React.FC = () => {
    return (
        <tr className="table-row">
            <th className="table-row__cell table-row__cell--header">
                Name
            </th>
            <th colSpan={2} className="table-row__cell table-row__cell--header">
                Status
            </th>
        </tr>
    )
}

export default TableHeader