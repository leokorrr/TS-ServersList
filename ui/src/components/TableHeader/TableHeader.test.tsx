import React from 'react'
import TableHeader from './TableHeader'
import { shallow, configure } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

configure({ adapter: new Adapter() })
it("Table Header rendered without crashing", () => {
    shallow(<TableHeader />);
})
it("should render initial layout", () => {
    const component = shallow(<TableHeader />);
    expect(component.getElements()).toMatchSnapshot();
})