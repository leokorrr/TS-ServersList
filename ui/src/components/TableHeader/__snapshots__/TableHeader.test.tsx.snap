// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render initial layout 1`] = `
Array [
  <tr
    className="table-row"
  >
    <th
      className="table-row__cell table-row__cell--header"
    >
      Name
    </th>
    <th
      className="table-row__cell table-row__cell--header"
      colSpan={2}
    >
      Status
    </th>
  </tr>,
]
`;
